spec/javascripts/controllers/toolbar_controller_spec.js

```diff
@@ -0,0 +1,79 @@
+describe('toolbarController', function () {
+  beforeEach(module('ManageIQ.toolbar'));
+
+  var middleware_toolbar_list = getJSONFixture('toolbar_middleware_server_list.json');
+  var middleware_toolbar_detail = getJSONFixture('toolbar_middleware_server_detail.json');
+
+  var responseData;
+  var $controller, $httpBackend, $scope, $q;
+
+  beforeEach(inject(function($injector) {
+     $q = $injector.get('$q')
```

Comment:
Yeah, Atom thought it would be better to use 3-spaces instead of 2. Will fix it down to 2.